lib/Net/Google/Drive/Simple.pm

```diff
@@ -262,6 +262,45 @@ sub file_upload {
     return $file_id;
 }
 
+###########################################
+sub rename {
+###########################################
+    my ( $self, $file_id, $new_name ) = @_;
+
+    my $url = URI->new( $self->{api_file_url} . "/$file_id" );
+
+    my $req = &HTTP::Request::Common::PUT(
```

Comment:
The `&` is not required there
`s{&HTTP::Request::Common::PUT}{HTTP::Request::Common::PUT}`
... I see you copy/paste it from another function